Resources/views/standard_layout.html.twig

```diff
@@ -67,7 +67,8 @@ file that was distributed with this source code.
             {% set locale = app.request.locale %}
             {# localize moment #}
             {% if locale[:2] != 'en' %}
-                <script src="{{ asset('bundles/sonatacore/vendor/moment/locale/' ~ locale|replace({'_':'-'}) ~ '.js') }}"></script>
+                {% set locale_lower = locale|lower %}
+                <script src="{{ asset('bundles/sonatacore/vendor/moment/locale/' ~ locale_lower|replace({'_':'-'}) ~ '.js') }}"></script>
```

Comment:
Any tips on how to linebreak? I will need to add ´{% spaceless %}´ to the script call.